examples/notebooks/CheckTransforms.ipynb

```diff
@@ -3,11 +3,7 @@
   {
```

Comment:
Please remember me, how you solve issue with bboxes out of image shape

Reply:
Check function `DetectionDataset.filter_bboxes' in torchok/data/datasets/detection/detection.py